include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -118,6 +118,13 @@ class BlmcJointModule
      */
     void set_zero_angle(const double& zero_angle);
 
+    /**
+     * @brief reverse the joint polarity rotation axis
+     * 
+     * @param reverse_polarity true:reverse rotation axis, false:do nothing
+     */
+    void reverse_joint_polarity(const bool& reverse_polarity);
```

Comment:
call this `toggle_joint_polartiy`? "Toggle" is what is used in software usually.

Reply:
Should there also be a `set_joint_polarity(const bool& should_reverse)`?

Reply:
I actually like the `set_joint_polarity(const bool& should_reverse)` I will to that in both classes